resources/js/pages/public/index.tsx

```
@@ -0,0 +1,185 @@
+import { Head } from '@inertiajs/react';
+import { Button } from '@/components/ui/button';
+import { Card, CardContent, CardHeader } from '@/components/ui/card';
+import AppLogoIcon from '@/components/app-logo-icon';
```

ESLint

[error] `@/components/app-logo-icon` import should occur before import of `@/components/ui/button`

(import/order)